file.go

```diff
@@ -12,13 +12,20 @@ import (
 
 // File contains the structures of a parsed WIRE File.
 type File struct {
+	isIncoming     bool           `json:"-"`
 	ID             string         `json:"id"`
 	FEDWireMessage FEDWireMessage `json:"fedWireMessage"`
```

Comment:
It's a common pattern to put private fields below exported fields. 

```suggestion
	ID             string         `json:"id"`
	FEDWireMessage FEDWireMessage `json:"fedWireMessage"`
	
	isIncoming     bool           `json:"-"`	
```